schema.graphql

```diff
@@ -182,42 +186,44 @@ type ExternalErc20Award @entity {
   symbol: String!
   decimals: BigInt
   
-  prizeStrategy: PrizeStrategy
+  prizeStrategy: SingleRandomWinner
```

Comment:
I'm not sure how this will work with multiple prize strategy types. Can we define in the GraphQL schema something like:
`prizeStrategy: SingleRandomWinner || MultipleRandomWinner || OtherPrizeStrat` ?

Reply:
If anything, I'd like to see us pull out the address/name/symbol/decimals into an ERC20Entity and turn this into a join table between SingleRandomWinner and ERC20Entity.  That way we can add more strategies down the line just by adding more join tables